meerk40t/gui/scene/scenewidgets.py

```diff
@@ -1,3 +1,4 @@
+from cmath import pi
```

Comment:
I don't really know the reason to import this here you import math just below this and use `math.pi` so this isn't used and likely should just use the regular math library.

Reply:
This is the too smart  for its own good Visual Code Studio feature to look for undeclared stuff and add imports on its own...